endtoend-testing/pom.xml

```diff
@@ -16,9 +16,8 @@
         </dependency>
         <dependency>
             <groupId>de.jplag</groupId>
-            <artifactId>java</artifactId>
+            <artifactId>cli</artifactId>
```

Comment:
I changed it to CLI as the end2end tests relied on the language loader in order to get the language. However, this approach made it pretty hard to understand that in order to support a new language, the according dependency has to be added to the pom.xml. Additionally, this made the end2end test description dependent on the language identifiers. Thus, if they changed in the future, it would break the tests.
Thus, I changed it back to `java` and added a small utility class to load the language from the identifier based on statically typed identifiers.

Reply:
Somewhere we should test the functions of the language loader (from my point of view) esp. if we have E2E tests :D

Reply:
There are some tests for the language loader in the CLI package